package.json

```diff
@@ -33,10 +33,9 @@
     "cssnano": "3.10.0",
     "del": "3.0.0",
     "eslint-config-rhinogram": "0.0.10",
-    "gulp": "3.9.1",
+    "gulp": "^3.9.1",
```

Comment:
We prefer to lock down our dependencies to specific versions, so here remove the carets (^).

Reply:
Haha, was testing to make sure changes were coming through. Ill fix that right quick